extensions/schema-compare/src/schemaCompareResult.ts

```diff
@@ -217,6 +234,16 @@ export class SchemaCompareResult {
 		this.compareButton.enabled = true;
 		this.optionsButton.enabled = true;
 
+		// explicitly exclude things that were excluded in previous compare
+		let thingsToExclude = this.sourceTargetSwitched ? this.originalTargetExcludes : this.originalSourceExcludes;
```

Comment:
const

Reply:
for my understanding - why const?

Reply:
nitpick, usually if we are not updating a variable after its initialization, usually mark it as const, this can avoid it being updated mistakenly.

Reply:
Even for a function scoped one? (will do).